package.json

```diff
@@ -4,6 +4,11 @@
   "private": false,
   "homepage": "https://linkfree.eddiehub.org",
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@mui/icons-material": "^5.10.6",
+    "@mui/material": "^5.10.7",
+    "@reduxjs/toolkit": "^1.8.5",
```

Comment:
I think redux is too much just for handling dark mode. If we need any global state then context should be considered first if it can’t be done with just prime react components

Reply:
But to add the theme changes across multiple pages simultaneously a global state is required, hence i think there is a need to use redux.

Reply:
Or context or just local storage. I believe Redux is too much when we’re not handling any other global state.